redwood-dom-testing/src/commonMain/kotlin/app/cash/redwood/dom/testing/DomSnapshotter.kt

```diff
@@ -40,9 +50,75 @@ public class DomSnapshotter @PublishedApi internal constructor(
       },
     ).await()
 
-    snapshotStore.put("$path/${name ?: "snapshot"}.png", image)
+    val fileName = "$path/${name ?: "snapshot"}.png"
```

Comment:
This is unrelated to your PR, but was changed to be non-null in Jesse's PR and I guess we missed updating this reference.
```suggestion
    val fileName = "$path/$name.png"
```